packages/neutron-sdk/src/interchain_queries/v045/types.rs

```diff
@@ -388,6 +394,54 @@ impl KVReconstruct for GovernmentProposal {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+/// Proposal vote option defines the members of a governance proposal vote option.
+pub struct WeightedVoteOption {
+    pub option: i32,
+    pub weight: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+/// Proposal vote defines the core field members of a governance proposal votes.
+pub struct ProposalVote {
+    pub proposal_id: u64,
+    pub voter: String,
+    pub options: Vec<WeightedVoteOption>,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+/// A structure that can be reconstructed from **StorageValues**'s for the **Government Proposal Votes Interchain Query**.
+pub struct GovernmentProposalVotes {
+    pub proposal_votes: Vec<ProposalVote>,
+}
+
+impl KVReconstruct for GovernmentProposalVotes {
+    fn reconstruct(storage_values: &[StorageValue]) -> NeutronResult<GovernmentProposalVotes> {
+        let mut proposal_votes = Vec::with_capacity(storage_values.len());
+
+        for kv in storage_values {
+            let voter_vote: Vote = Vote::decode(kv.value.as_slice())?;
```

Comment:
what if kv.value is empty?

Reply:
in this case it will return empty vote with zero valued fields. Do you think it is better to filter it out?

Reply:
oh. my bad. just checked it and the parsing works fine in this case 👍